pick.lic

```diff
@@ -708,48 +714,104 @@ class Pick
 
   def loot(box_noun)
     echo "Looting #{box_noun}..." if @debug
-    if DRC.bput("open my #{box_noun}", /You open/, /^In the .* you see .*\./, 'That is already open', 'It is locked') == 'It is locked'
+    
+    if DRC.bput("open my #{box_noun}", /^In the .* you see .*\./, 'That is already open', 'It is locked') == 'It is locked'
       echo 'Bug: Tried to loot locked box...'
       return
     end
-
-    if (@settings.fill_pouch_with_box || @settings.loot_specials.empty?)
+		
+    #pouch full, fill from lootpouch then try again      	
+    if Flags['full-lootpouch']
+      Flags.reset('full-lootpouch')
+      echo 'last match:, #{Regexp.last_match}'	 
+      echo 'last match0:, #{Regexp.last_match(0)}'	 
+      echo 'last match1:, #{Regexp.last_match(1)}'	       
+      case DRC.bput("fill my #{@settings.gem_pouch_adjective} #{@settings.gem_pouch_noun} with my #{@settings.autoloot_container}", /You open/, /is too full to fit another gem/, /is too full to fit any more gems/, /You'd better tie it up before putting/, /You'll need to tie it up before/)        
```

Comment:
💡  This looks like a good candidate for a new method in `common-items`, maybe `DRCI.fill_container(source_container, target_container)`

Reply:
I used the pattern that was already present and could be updated by someone. I don't wish to extend other units just yet.

Reply:
@KatoakDR Created https://github.com/rpherbig/dr-scripts/pull/5879 for this :)